src/styles/_catalog.scss

```
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 0.3s;
```

Use full transition rule and list styles that you apply it to.
Also, use variable for transition duration.
```suggestion
    transition: transform 0.3s, color 0.3s;
```